float64 desiredYaw
float64 actualYaw
float64 yawError
float64 yawCommand
float64 yawCommandLimited
